src/ray/raylet/raylet.h

```diff
@@ -16,7 +16,7 @@ namespace ray {
 
 namespace raylet {
 
-using rpc::ClientTableData;
```

Comment:
I don't think it is a good practice to use `using xxx;` in a header file. It might pollute the namespace. 

You added an unique prefix `Gcs` to the name, but I don't think it's an appropriate way.
There is no difference between `GcsNodeInfo`  and `gcs::NodeInfo`.

Reply:
Agreed that we should use namespace of ray project directly rather than `using xxx;`. 
Maybe we can do it from now on and refactor old codes in a new PR.

En, I prefer gcs::NodeInfo and  @raulchen is prefer GcsNodeInfo.

Reply:
Sounds good.